sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/aws.yaml

```diff
@@ -143,7 +143,7 @@ provider:
             "A1XJw=="
           containsSecret: true
           match: |
-            "IQoJb3JpZ2luX2VjEKL//////////wE" +
+            IQoJb3JpZ2luX2VjEKL//////////wE" +
```

Comment:
Pattern contains opening quote in a non-captured group, so the quote sign is not actually present in the match